web/extensions/core/editAttention.js

```diff
@@ -57,8 +68,9 @@ name:id,
         };
 
         function editAttention(event) {
-            const inputField = event.composedPath()[0];
-            const delta = 0.025;
+            const inputField = event.target;
```

Comment:
`event.composedPath()[0];` is equivalent `event.target` in most cases, but is also just straight up better as if there is no shadow dom it returns `event.target` and if there is it returns the original target which is what we want. While comfy doesn't have a shadow dom by default, other frontends may add it. And not to look at the original code too much either but they do `event.target || event.composedPath()[0]` for a reason. However, the or is not necessary, `event.composedPath()[0]` is all you need and is better in more cases.

```suggestion
            const inputField = event.composedPath()[0];
```